### PR TITLE
Update Glossary

### PR DESCRIPTION
**Pull request by corinnaguenther**
[#49185] Keep Glossary updated

https://community.openproject.org/work_packages/49185

- add: Work and Remaining work
- edit: Gantt charts
- add: Project lists
- edit: Quick context menu

### CODE REVIEW

File: docs/glossary/README.md [hunk 1]
```diff
@@ -392,6 +396,10 @@ In OpenProject, a wiki is defined as a [module](#module) that allows to use wiki
 
 ![A wiki module in OpenProject](glossary-openproject-wiki.png)
 
+### Work and Remaining Work
+
+Work is defined in OpenProject as a work package attribute that indicates in hours how much work is estimated to be required to complete a work package. You can edit the value for work manually, as well as the value for the project attribute '**Remaining work**'. To make it easier for project managers to work with work package hierarchies, OpenProject also displays a value (in blue) for the total amount of work in the Work field for parent work packages – next to the value for the dedicated work package. This **total work value** is the sum of the work value of the parent work package and all the work values of its children. The same principle applies to the work package attribute Remaining Work. [Read in our user guide about how to configure a work package](../user-guide/work-packages/work-package-table-configuration/)
```
Comment (aaron-contreras) [inline docs/glossary/README.md]:
```suggestion
Work is defined in OpenProject as a work package attribute that indicates in hours how much work is estimated to be required to complete a work package. You can edit the value for work manually, as well as the value for the work package attribute '**Remaining work**'. To make it easier for project managers to work with work package hierarchies, OpenProject also displays a value (in blue) for the total amount of work in the Work field for parent work packages – next to the value for the dedicated work package. This **total work value** is the sum of the work value of the parent work package and all the work values of its children. The same principle applies to the work package attribute Remaining Work. [Read in our user guide about how to configure a work package](../user-guide/work-packages/work-package-table-configuration/)
```

File: docs/glossary/README.md [hunk 2]
```diff
@@ -320,7 +324,7 @@ In OpenProject, projects can be private or public. Public means that the project
 
 ### Quick context menu
 
-The quick context menu in OpenProject opens when you open a [work package table](#work-package-table) and then right-click on a single work package. It allows you to perform quick actions like *open details view*, *copy link to clipboard* or *create new child*. Please note that the quick context menu differs depending on whether you are in the [Gantt](#gantt-chart) view or not.
+The quick context menu in OpenProject opens when you open a [work package table](#work-package-table) and then right-click on a single work package. It allows you to perform quick actions like *open details view*, *copy link to clipboard* or *create new child*. Please note that the quick context menu differs depending on whether you are in the [Gantt](#gantt-chart) module or not.
```
Comment (oliverguenther) [inline docs/glossary/README.md]:
```suggestion
The quick context menu in OpenProject opens when you open a [work package table](#work-package-table) and then right-click on a single work package. It allows you to perform quick actions like *open details view*, *copy link to clipboard* or *create new child*. Please note that the quick context menu differs depending on whether you are in the [Gantt charts](#gantt-chart) module or not.
```

File: docs/glossary/README.md [hunk 3]
```diff
@@ -392,6 +396,10 @@ In OpenProject, a wiki is defined as a [module](#module) that allows to use wiki
 
 ![A wiki module in OpenProject](glossary-openproject-wiki.png)
 
+### Work and Remaining Work
+
+Work is defined in OpenProject as a work package attribute that indicates in hours how much work is estimated to be required to complete a work package. You can edit the value for work manually, as well as the value for the work package attribute '**Remaining work**'. To make it easier for project managers to work with work package hierarchies, OpenProject also displays a value (in blue) for the total amount of work in the Work field for parent work packages – next to the value for the dedicated work package. This **total work value** is the sum of the work value of the parent work package and all the work values of its children. The same principle applies to the work package attribute Remaining Work. [Read in our user guide about how to configure a work package](../user-guide/work-packages/work-package-table-configuration/)
```
Comment (MayaBerd) [inline docs/glossary/README.md]:
```suggestion
In OpenProject, 'Work' refers to a work package attribute indicating the estimated hours needed to complete a task. You can edit the value for work manually, as well as the value for the work package attribute '**Remaining work**'. To make it easier for project managers to work with work package hierarchies, OpenProject also displays a value (in blue) for the total amount of work in the Work field for parent work packages – next to the value for the dedicated work package. This **total work value** is the sum of the work value of the parent work package and all the work values of its children. The same principle applies to the work package attribute Remaining Work. [Read in our user guide about how to configure a work package](../user-guide/work-packages/work-package-table-configuration/)
```

File: docs/glossary/README.md [hunk 4]
```diff
@@ -292,6 +292,10 @@ Project folders help collaborating in the most efficient way. They can be used w
 
 The project identifier is defined as the unique name used to identify and reference projects in the application as well as in the address bar of your browser. Project identifiers can be changed in the [project settings](#project-settings).
 
+### Project lists
+
+In OpenProject, project lists are very useful for project portfolio managers to get an overview of all their [projects](#project) on the instance. Access your project lists on OpenProject by either navigating to the "select a project" menu and clicking on the "Project list" button, or via the [Global modules](#global-modules). [Read more about project lists OpenProject](../user-guide/projects/projects-lists/).
```
Comment (MayaBerd) [inline docs/glossary/README.md]:
```suggestion
In OpenProject, project lists are very useful for project portfolio managers to get an overview of all their [projects](#project) on the instance. Access your project lists on OpenProject by either navigating to the "select a project" menu and clicking on the "Project list" button, or via the [Global modules](#global-modules). [Read more about project lists OpenProject](../user-guide/projects/project-lists/).
```

**Review by MayaBerd — COMMENTED** [review]
@corinnaguenther I've added my small suggestions. 
I've corrected the link for "Project lists" directly, I will be adjusting it in the rest of documentation today.

**Pull request merged**